conjure_oxide/tests/generated_tests.rs

```diff
@@ -66,4 +70,57 @@ fn integration_test(path: &str, essence_base: &str) -> Result<(), Box<dyn Error>
     Ok(())
 }
 
+/// Recursively sorts the keys of all JSON objects within the provided JSON value.
+///
+/// serde_json will output JSON objects in an arbitrary key order.
+/// this is normally fine, except in our use case we wouldn't want to update the expected output again and again.
+/// so a consistent (sorted) ordering of the keys is desirable.
+///
+/// sorting is done in-place.
+fn sort_json_object(value: &Value) -> Value {
```

Comment:
This doesn't do what you are asking for here, but you can enable the `preserve_order` feature of serde_json, which will make it keep the same order as when you read in a json file.

Reply:
You may be right. The main thing I observed and was testing for is the ordering of the vector of variables. I suspect the serde option won't do anything about that, right?

I'll edit when I am at the computer.

Reply:
This is a json value we are generating using serde. I think the in-place comment was wrong (left over from an earlier experiment) but the rest does sort the keys I believe. Happy to be shown otherwise, I am also new in this game!